__init__.py

```diff
@@ -1,22 +1,24 @@
-import importlib
-import os
-
-node_list = [ #Add list of .py files containing nodes here
-    "advanced_model_merging",
-    "reference_only",
-    "sampler_rescalecfg",
-    "sampler_tonemap",
-    "sdxl_model_merging"
-]
-
-NODE_CLASS_MAPPINGS = {}
-NODE_DISPLAY_NAME_MAPPINGS = {}
-
-for module_name in node_list:
-    imported_module = importlib.import_module(".{}".format(module_name), __name__)
-
-    NODE_CLASS_MAPPINGS = {**NODE_CLASS_MAPPINGS, **imported_module.NODE_CLASS_MAPPINGS}
-    if hasattr(imported_module, "NODE_DISPLAY_NAME_MAPPINGS"):
-        NODE_DISPLAY_NAME_MAPPINGS = {**NODE_DISPLAY_NAME_MAPPINGS, **imported_module.NODE_DISPLAY_NAME_MAPPINGS}
-
-__all__ = ['NODE_CLASS_MAPPINGS', 'NODE_DISPLAY_NAME_MAPPINGS']
+import importlib
+import os

```

Comment:
Why did this entire file change? Did you change the line endings?

Reply:
Could be vscode. Happens with some PRs from people. Restructures everything cause they have pretty or whatever it is installed for their projects.

Reply:
yeah that should be what happened since I'm using VSCode